internal/ir/xds.go

```diff
@@ -490,6 +490,8 @@ type HTTPRoute struct {
 	TCPKeepalive *TCPKeepalive `json:"tcpKeepalive,omitempty" yaml:"tcpKeepalive,omitempty"`
 	// Retry settings
 	Retry *Retry `json:"retry,omitempty" yaml:"retry,omitempty"`
+	// EnvoyExtensionFeatures
```

Comment:
prefer eliminating 1 level of hierarchy in the IR
* adding `ExtProcs []ExtProc` here
* flattening Service and adding those fields into `ExtProc`